cli/admin/README.md

```diff
@@ -126,5 +126,66 @@ Note that if you are using the standalone deployment, Bookkeeper commands (and o
 available. For this reason, we encourage you to go a step further and deploy a full Pravega cluster to 
 explore the whole functionality of Pravega Admin CLI.
 
+## Pravega CLI on Kubernetes
+
+The admin level pravega-cli needs to be accessed from a pod that has access to the bookkeeper and zookeeper such as the segmentstore pod.
+
+Once in the pod, you can run the pravega-cli under admin mode
+```
+    ./bin/pravega-admin
+        OpenJDK 64-Bit Server VM warning: Option MaxRAMFraction was deprecated in version 10.0 and will likely be removed in a future release.
+        Pravega Admin CLI.
+        
+        Exception reading input properties file: null
+        Initial configuration:
+            pravegaservice.container.count=4
+            bookkeeper.ledger.path=/pravega/pravega/bookkeeper/ledgers
+            cli.security.auth.enable=false
+            cli.security.auth.credentials.password=1111_aaaa
+            pravegaservice.cluster.name=pravega/pravega
+            cli.store.metadata.backend=segmentstore
+            cli.controller.rest.uri=http://localhost:9091
+            cli.security.auth.credentials.username=admin
+            pravegaservice.zk.connect.uri=localhost:2181
+            cli.controller.grpc.uri=tcp://localhost:9090
+```
+
+The initial configuration needs to be modified by accessing the properties file (./conf/admin-cli.properties) or using `config set` command.
+For example:
+```
+config set cli.controller.rest.uri=pravega-pravega-controller.default:10080
+
+config list
+    pravegaservice.container.count=4
+    bookkeeper.ledger.path=/pravega/pravega/bookkeeper/ledgers
+    cli.security.auth.enable=false
+    cli.security.auth.credentials.password=1111_aaaa
+    pravegaservice.cluster.name=pravega/pravega
+    cli.store.metadata.backend=segmentstore
+    cli.controller.rest.uri=pravega-pravega-controller.default:10080
+    cli.security.auth.credentials.username=admin
+    pravegaservice.zk.connect.uri=localhost:2181
+    cli.controller.grpc.uri=tcp://localhost:9090
+```
+
+
+To modify the configuration file access the beginning of segmentstore logs to get the current configuration.
+```
+kubectl logs pravega-pravega-segment-store-0
```

Comment:
Maybe we could add here the `grep` as well to the parameters, like:
`kubectl logs pravega-pravega-segment-store-0 | grep -i <parameter-to-look-for>`
Please, check that this works with the parameters posted below.